src/components/renderer/form-list-table.vue

```diff
@@ -327,4 +357,29 @@ export default {
 .btn-outline-secondary {
   border: none;
 }
+
+.item-text {
+  color: #42526E;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -0.02em;
+  text-align: left;
+}
+</style>
+
+<style lang="scss">
```

Comment:
other?

Reply:
the first one is scoped, and the b-dropdown doesn't use styles